donation-api/src/main/java/org/interfaithsanctuary/donationapi/controller/GiftController.java

```diff
@@ -3,21 +3,61 @@
 import org.interfaithsanctuary.donationapi.model.Gift;
 import org.interfaithsanctuary.donationapi.repository.GiftRepository;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.Optional;
 
 @RestController
 @RequestMapping(path="/gifts")
 public class GiftController {
+
     @Autowired
     private GiftRepository giftRepository;
 
-    @RequestMapping(method = RequestMethod.GET)
-    public Iterable<Gift> getAllGifts() {
+    @GetMapping("/all")
+    public @ResponseBody
```

Comment:
@ ResponseBody isn't needed because the @ RestController will always add it to each controller method. Not a huge deal though!